components/sync15/src/clients/engine.rs

```diff
@@ -73,7 +73,16 @@ impl<'a> Driver<'a> {
             // clients collection, so we don't check for `is_tombstone`.
             // https://github.com/mozilla/application-services/issues/1801
             // tracks deleting these from the server.
-            let client: ClientRecord = payload.into_record()?;
+            if payload.is_tombstone() {
+                continue;
```

Comment:
it probably makes sense to log that we ignored a payload here?

Reply:
Yup, that sounds good; I'll add some `debug` log lines for when we drop them